libcaf_core/caf/logger.hpp

```diff
@@ -112,14 +112,17 @@ class logger : public ref_counted {
 
     event& operator=(const event&) = default;
 
-    event(int lvl, string_view cat, string_view full_fun, string_view fun,
-          string_view fn, int line, std::string msg, std::thread::id t,
+    event(unsigned lvl, unsigned line, string_view cat, string_view full_fun,
+          string_view fun, string_view fn, std::string msg, std::thread::id t,
           actor_id a, timestamp ts);
 
     // -- member variables -----------------------------------------------------
 
     /// Level/priority of the event.
-    int level;
+    unsigned level;
```

Comment:
The function `accepts` still expects the level as an int.

Reply:
Good catch. Fixed.